widgetssdk/src/main/java/com/glia/widgets/entrywidget/adapter/EntryWidgetAdapter.kt

```diff
@@ -38,6 +39,20 @@ internal class EntryWidgetAdapter(
         entryWidgetTheme?.errorButton
     )
 
+    constructor(
+        viewType: EntryWidgetContract.ViewType,
+        entryWidgetTheme: SecureMessagingTheme?
+    ) : this(
+        viewType,
+        entryWidgetTheme?.mediaTypeItems,
+        null, // TODO: is this expected? I don't know where to get error colors
```

Comment:
Is this expected to be merged?

Reply:
I got the case, I think this is expected, because for SC chat banner we don't have error state

Reply:
No, I forgot about that.
Anyway, I wanted to discuss this topic in this PR. I am really not sure what theme to apply here. This theme settings are for the EntryWidgetsView error state (errorTitle, errorMessage, errorButton) but in chat.secureMessaging remote theme there is nothing I can use for this.

Reply:
Yes, because in the chat screen, we don't have the error state, in case of an error, the top banner should disappear.